x-pack/test_serverless/functional/test_suites/search/config.ts

```diff
@@ -18,4 +18,8 @@ export default createTestConfig({
   // include settings from project controller
   // https://github.com/elastic/project-controller/blob/main/internal/project/esproject/config/elasticsearch.yml
   esServerArgs: [],
+  kbnServerArgs: [
```

Comment:
Same question here, this seems like an unintentional commit

Reply:
same answer as above, needed to properly test the endpoints flyout in an FTR test.